isis/src/qisis/objs/Directory/ImportControlNetWorkOrder.cpp

```diff
@@ -206,32 +207,12 @@ namespace Isis {
     }
     m_status = WorkOrderFinished;
     m_readProgresses.clear();
-  }
-
 
-  /**
-   * @brief Deletes the control network
-   *
-   * This method deletes the control network from the project. This method is was
-   * renamed from undoSyncRedo() to undoExecution().
-   */
-  void ImportControlNetWorkOrder::undoExecution() {
-    if (m_list && project()->controls().size() > 0 && m_watcher->isFinished()) {
-      // Remove the controls from disk.
-      m_list->deleteFromDisk( project() );
-      // Remove the controls from the model, which updates the tree view.
-      ProjectItem *currentItem =
-          project()->directory()->model()->findItemData( QVariant::fromValue(m_list) );
-      project()->directory()->model()->removeItem(currentItem);
-
-      foreach (Control *control, *m_list) {
-        delete control;
-      }
-      delete m_list;
-    }
+    // If one control network was imported, then activeControl() will set the
+    // active control to that control network
+    project()->activeControl();
```

Comment:
I would just add a bit more to the comment, such as "If one control network was imported, then activeControl() will set the newly imported control net to the active if project does not curently have an active control set and there are no other control nets in the project."